### ISSUE TITLE
Training Details for Detector

### ISSUE THREAD

**Issue by DeanChan** [phil-bergmann/tracking_wo_bnw#70]
Hi there~ Many thanks for your work!

I have some questions regarding the training script in [this colab script](https://colab.research.google.com/drive/1_arNo-81SnqfbdtAhb3TBSU5H0JXQ0_1).

Looks like the detector is first initialized from the coco pre-trained weights:
```python
# load an instance segmentation model pre-trained on COCO
model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained=True)
```
Then it's overridden by a pre-trained model on MOT:
```python
model_state_dict = torch.load(f"gdrive/My Drive/Colab Notebooks/faster_rcnn_fpn_training_mot_17/resnet50/model_epoch_27.model")
model.load_state_dict(model_state_dict)
```
and the learning rate is set to an *unnaturally small* number:
```python
optimizer = torch.optim.SGD(params, lr=0.00001,
                            momentum=0.9, weight_decay=0.0005)
```

So this colab script is just for demonstration right? Could you provide the training details (e.g. parameter configs) for your actual training process? 

Thank you.

**Comment by timmeinhardt** [comment]
Yes this script is for demonstration purposes but the final model was trained with script. The original learning rate was `0.001`, if I remember correctly. The learning rate scheduler then decreases the learning rate after 10 epochs.

Sorry for not directly sharing the notebook in a cleaner way. We figured that for MOT17 one will use our trained model and retraining the detector on an entirely new dataset requires some form of hyperparameter optimization anyway.

**Comment by DeanChan** [comment]
Thank you for your reply!

So the original training script is almost identical to the current colab script, except for removing the pretrained `model_epoch_27.model` checkpoint and change the learning rate from `0.00001` to `0.001`. Is that correct?

**Comment by timmeinhardt** [comment]
Yes, this should be really close to the initial training configuration..

**Comment by DeanChan** [comment]
Thanks a lot!
